easybuild/easyconfigs/p/PsiCLASS/PsiCLASS-1.0.3-GCC-11.2.0.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = "MakeCp"
```

Comment:
@deniskristak It's probably cleaner to use `Binary` as easyblock here, since you're just copying everything anyway...

Or, if that works, you should refine `files_to_copy` to only the binaries are copied, and copy them into a `bin` subdirectory?

Reply:
@boegel to be honest, whole directory is only 15 MB and I am a bit afraid that I might forget to copy something important which might show later...therefore, wouldn't it be better to just copy everything here?

Reply:
OK, I guess it makes sense to stick to `MakeCp` with that in mind...